frappe/desk/page/setup_wizard/install_fixtures.py

```diff
@@ -8,16 +8,25 @@
 from frappe.desk.doctype.global_search_settings.global_search_settings import update_global_search_doctypes
 
 def install():
-	update_genders_and_salutations()
+	update_genders()
+	update_salutations()
 	update_global_search_doctypes()
 	setup_email_linking()
 
 @frappe.whitelist()
-def update_genders_and_salutations():
-	default_genders = [_("Male"), _("Female"), _("Other")]
-	default_salutations = [_("Mr"), _("Ms"), _('Mx'), _("Dr"), _("Mrs"), _("Madam"), _("Miss"), _("Master"), _("Prof")]
+def update_genders():
+	default_genders = [_("Male"), _("Female"), _("Other"), _("Trans Man"), _("Trans Woman"), _("Transgender"), _("Genderqueer"),
```

Comment:
Change genders to [_("Male"), _("Female"), _("Other"), _("Transgender"), _("Genderqueer"), _("Non-Conforming"), _("Prefer not to say")]